coralproject/cay
# Publish gallery

## What does this PR do?

Once complete, this PR will enable publishing a gallery to s3, previewing and embed code.

fixes #440 

## How do I test this PR?

- must have the `publish-gallery` branch running on Elkhorn also https://github.com/coralproject/elkhorn/pull/59
- open a gallery of your choice
- click `Get embed codes`. Do the correct codes come up?
- click `Preview` on the right. Does the panel open up and can you see the gallery preview?
- can you reach the iframe in the embed code?
- enter a title and description. Do they show up on the preview?
- does the PII show up in the right place (above/below) depending on what you selected?
- do the right PII elements show up based on what was selected?

@coralproject/frontend
